samples/sampler/stories/default/TooltipDecorator.js

```diff
@@ -39,7 +40,7 @@ export default {
 };
 
 export const _TooltipButton = () => (
-	<div style={{textAlign: 'center'}}>
+	<div style={{marginTop: ri.scaleToRem(99), textAlign: 'center'}}>
```

Comment:
I don't think it's bad to overlap with the text on the back.
so how about not fix this